app/locale/locale.coffee

```diff
@@ -152,3 +154,4 @@ Object.defineProperties module.exports,
         return
 
       Vue.use(VueI18Next)
+      moduleVue.use(VueI18Next)
```

Comment:
What is `moduleVue`?

Reply:
Removed this - it was leftover from debugging a Vue issue I had